src/server/pachw/server/env.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"context"
+	"path"
+
+	"github.com/sirupsen/logrus"
+	etcd "go.etcd.io/etcd/client/v3"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/serviceenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/task"
+)
+
+// Env is the dependencies needed to run the pachW Master
+type Env struct {
+	EtcdPrefix        string
+	EtcdClient        *etcd.Client
+	TaskService       task.Service
+	KubeClient        kubernetes.Interface
+	Namespace         string
+	Enabled           bool
```

Comment:
Is this field needed?

Reply:
Good catch, yeah I don't think its actually used by the pachw code

Reply:
Actually, I use this in the builder code to determine whether to spin up a master or not in maybeInitPachwController(). For some reason my IDE was showing me 0 usages.

Reply:
So instead of using enabled, I can just check the pachwEnabled flag on the Env struct.